### PR TITLE
IWE-39 -- BE: Create required migrations based upon IWE-10

### PR DESCRIPTION
**Pull request by liadGolan**
### `IWE-39` - `PR`
`**READY**`

### Migrations
`YES `

### What's this PR do?
`Change the migrations for the application`
### Where should the reviewer start?  
`Make sure the migrations folder reflects the changes specified in IWE-10`

### Related PRs

IWE-10 

### Requirements & Dependencies
no
### Impacted Areas in Application
Most of the tables will be changed or removed

### CODE REVIEW

File: database/migrations/2016_04_06_120156_create_funds_table.php [hunk 1]
```diff
@@ -1,36 +0,0 @@
-<?php
```
Comment (alejandrom2) [inline database/migrations/2016_04_06_120156_create_funds_table.php]: Need

File: database/migrations/2016_04_18_115024_create_invitations_table.php [hunk 2]
```diff
@@ -1,39 +0,0 @@
-<?php
```
Comment (alejandrom2) [inline database/migrations/2016_04_18_115024_create_invitations_table.php]: needed

File: database/migrations/2016_05_10_101444_create_permissions_table.php [hunk 3]
```diff
@@ -1,33 +0,0 @@
-<?php
```
Comment (alejandrom2) [inline database/migrations/2016_05_10_101444_create_permissions_table.php]: needed

File: database/migrations/2016_05_10_105431_create_permission_role_table.php [hunk 4]
```diff
@@ -1,32 +0,0 @@
-<?php
```
Comment (alejandrom2) [inline database/migrations/2016_05_10_105431_create_permission_role_table.php]: needed

File: database/migrations/2016_06_02_101355_create_visibilities_table.php [hunk 5]
```diff
@@ -1,32 +0,0 @@
-<?php
```
Comment (alejandrom2) [inline database/migrations/2016_06_02_101355_create_visibilities_table.php]: needed

File: database/migrations/2016_06_02_123422_create_policy_types_table.php [hunk 6]
```diff
@@ -1,32 +0,0 @@
-<?php
```
Comment (alejandrom2) [inline database/migrations/2016_06_02_123422_create_policy_types_table.php]: needed

File: database/migrations/2016_07_12_154854_add_start_end_funds_column.php [hunk 7]
```diff
@@ -1,34 +0,0 @@
-<?php
```
Comment (alejandrom2) [inline database/migrations/2016_07_12_154854_add_start_end_funds_column.php]: needed

File: database/migrations/2016_04_18_115024_create_invitations_table.php (hunk 2 above)
Comment (arthurwirsching) [reply database/migrations/2016_04_18_115024_create_invitations_table.php]: needed

**Pull request merged**